pkg/controller/core/clusterqueue_controller.go

```diff
@@ -341,7 +341,9 @@ func (r *ClusterQueueReconciler) Delete(e event.TypedDeleteEvent[*kueue.ClusterQ
 	defer r.notifyWatchers(e.Object, nil)
 
 	r.log.V(2).Info("ClusterQueue delete event", "clusterQueue", klog.KObj(e.Object))
-	r.cache.DeleteClusterQueue(e.Object)
+	if err := r.cache.DeleteClusterQueue(e.Object); err != nil {
+		r.log.Error(err, "Error deleting ClusterQueue from cache", "clusterQueue", klog.KObj(e.Object))
+	}
```

Comment:
@mimowo probably could use your input here about error handling / logging. The message could maybe be improved.

Reply:
This suggestion works, or just ignoring the error. I'm fine with either.